### PR TITLE
fix: remove cache from $http user, accouts request. 

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#354 

screen shot 2015-03-30 at 10 58 03 am
screen shot 2015-03-30 at 10 57 44 am

### CODE REVIEW

File: client/app/scripts/app.js [hunk 1]
```diff
@@ -215,10 +215,10 @@ angular
     $httpProvider.interceptors.push(function ($q, Auth, $location, localStorageService) {
       return {
         'request': function (config) {
-          if (!config.withoutToken) {
-            config.headers = {
+          if (!config.withoutToken && !config.url.match(/.html/)) {
+            angular.extend(config.headers, {
```
Comment (amitaibu) [inline client/app/scripts/app.js]: Question: wouldn't it better to move this interceptor to the auth.js service?

Comment (amitaibu) [inline client/app/scripts/app.js]: (could be done in follow up if needed, not urgent)

Comment (ceoaliongroo) [inline client/app/scripts/app.js]: No, the interceptors need to declare in the config phase.

I think we can take the auth.js and the interceptor and putting together in https://github.com/Gizra/angular-auth ,  as a standard angularjs module for authentication between drupal restful - angularjs.

note: using bower

File: client/app/scripts/services/auth.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ angular.module('negawattClientApp')
           'Authorization': 'Basic ' + Utils.Base64.encode(user.username + ':' + user.password)
         },
         withoutToken: true,
-        cache: true
+        cache: false
```
Comment (amitaibu) [inline client/app/scripts/services/auth.js]: remove ``cache``

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**